source/installation/install-with-docker.txt

```diff
@@ -0,0 +1,103 @@
+.. _rm-install-docker:
+
+===================
+Install with Docker
+===================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install and run Relational Migrator as a Docker container. 
+Running Relational Migrator with Docker ensures an installation process 
+that is both independent of the operating system and straightforward 
+to replicate.
+
+Before you Begin
+----------------
+
+Install `Docker <https://docs.docker.com/install/>`__
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Pull the MongoDB Docker Image
+      
+      .. code-block:: sh
+
+         docker pull public.ecr.aws/v4d7k6c9/relational-migrator:latest
+
+   .. step:: Run the Image as a Container
+
+      .. code-block:: sh
+
+         docker run -p 8080:8080 public.ecr.aws/v4d7k6c9/relational-migrator:latest
+
+      The ``-p 8080:8080`` in this command maps the container port to the host port.
+      This allows you to connect to Relational Migrator by browsing to 
+      `<http://localhost:8080>`__.
+
+      .. note:: Override a Config Setting
+
+         You can override a configuration setting by specifying the 
+         configuration setting in the Docker command.
```

Comment:
Possibly simplify: You can specify a configuration setting to override in the Docker command.